src/Integration.Vsix.UnitTests/Integration.Vsix.UnitTests.csproj

```diff
@@ -42,6 +42,34 @@
       <HintPath>..\..\packages\Microsoft.CodeAnalysis.Common.1.0.0\lib\net45\Microsoft.CodeAnalysis.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
I added them, they are required for the `Workspace`, but I'm not sure if all of them are needed or not. 

So, I'll check it tomorrow.